widget/view.html

```diff
@@ -3,62 +3,77 @@
+        data-uib-tooltip="Refresh"></span>
+    </div>
+  </div>
+  <div class="metric-widget-content" data-ng-hide="collapsed">
+    <div class="widget-cardview" data-ng-show="!processing">
+      <div class="metrics-container">
+        <div data-ng-init="loadChartData()">
+          <div class="connector-card-wrapper panel {{!collapse ? 'expand-class' : 'collapse-class'}}"
+            data-ng-repeat="item in configuredConnectorList" data-ng-if="item.configuration.length > 0">
+            <div class="connector-label-header">
+              <div class="col-md-12 padding-left-0 padding-right-0">
+                <div class="connector-card-header-text">
+                  <div class="connector-health-status display-flex-space-between" data-ng-click="collapse = !collapse">
+                    <div class="display-flex" >
+                      <div data-ng-if="item.icon_small">
+                        <img class="connector-logo-dimensions" data-ng-src="{{item.icon_small}}" alt="Base64 Image">
+                      </div>
+                      <div class="margin-left-18">
+                        <div class="text-overflow" title="{{item.label}}">{{item.label}}</div>
+                        <div class="connector-card-secondary-header-text"><span>{{item.configuration.length}}
+                            {{(item.configuration.length === 1)?'Configuration':'Configurations'}} Monitored</span></div>
+                      </div>
+                    </div>
+                    <div class="margin-left-auto;">
+                      <div class=" padding-right-sm">
+                        <span class="connector-button fa fa-lg"
+                          data-ng-class="{'fa-question-circle': item.connectorHealthStatus === 'unknown', 'fa-check-circle': item.connectorHealthStatus === 'available', 'fa-exclamation-circle': item.connectorHealthStatus === 'disconnected'}"></span>
+                        <span
+                          class="connector-health-status-message {{item.connectorHealthStatus}}">{{item.connectorHealthStatusMessage}}</span>
+                        <span class="fa config-collapse-btn"
+                          data-ng-class="{'fa-caret-down': collapse, 'fa-caret-up': !collapse}"></span>
+                      </div>
+                    </div>
                   </div>
                   <div class="connector-config-wrapper">
                     <div data-ng-hide="collapse" class="col-md-12 connector-config">
                       <div class="row margin-top-md">
                         <div class="col-md-6">
-                          <span class="pull-left muted font-10px">Configuration Name</span>
+                          <span class=" muted font-10px">Configuration Name</span>
                         </div>
                         <div class="col-md-6">
-                          <div class="pull-right">
+                          <div class="position-absolute right-10">
```

Comment:
position absolute not required here

Reply:
If position-absolute not used Right doesnt work, to stick it to right I've used position absolute